pom.xml

```diff
@@ -249,12 +240,6 @@
         <metadataModelApiVersion>${muleMetadataModelVersion}</metadataModelApiVersion>
 
         <!-- Test Extensions -->
-        <muleMetadataExtensionTestVersion>4.2.0-SNAPSHOT</muleMetadataExtensionTestVersion>
-        <muleVeganExtensionTestVersion>4.2.0-SNAPSHOT</muleVeganExtensionTestVersion>
-        <muleHeisenbergExtensionTestVersion>4.2.0-SNAPSHOT</muleHeisenbergExtensionTestVersion>
-        <muleMarvelExtensionTestVersion>4.2.0-SNAPSHOT</muleMarvelExtensionTestVersion>
-        <mulePetstoreExtensionTestVersion>4.2.0-SNAPSHOT</mulePetstoreExtensionTestVersion>
-        <muleSubtypesExtensionTestVersion>4.2.0-SNAPSHOT</muleSubtypesExtensionTestVersion>
```

Comment:
I'm pretty sure these are needed

Reply:
🤔 I'll validate again.

Reply:
I just double checked them and the only places where I see those properties being referenced are these:
```

| Propery Name                       | Usage                                                |
|------------------------------------|------------------------------------------------------|
| muleHeisenbergExtensionTestVersion | - mule/buildtools/scripts/update_deps_versions.sh:78 |
| muleSubtypesExtensionTestVersion   | - mule/buildtools/scripts/update_deps_versions.sh:79 |
| muleMarvelExtensionTestVersion     | - mule/buildtools/scripts/update_deps_versions.sh:80 |
| mulePetstoreExtensionTestVersion   | - mule/buildtools/scripts/update_deps_versions.sh:81 |
```

Which anyways means that are not necessary. However, I'll need to remove them also from `mule/buildtools/scripts/update_deps_versions.sh`

Nevertheless, thanks for highlighting this. It made me realize I might missed reviewing other files rather than `pom.xml`s on my analysis, forgetting the resource filtering part